src/modules/aggregator/controllers/aggregates/global/aggregator.global.character.aggregate.controller.ts

```diff
@@ -0,0 +1,19 @@
+import {BadRequestException, Controller} from '@nestjs/common';
+import {Ctx, MessagePattern, Payload, RmqContext} from '@nestjs/microservices';
+import AggregatorBaseController from '../../AggregatorBaseController';
+import {MQAcceptedPatterns} from '../../../../data/constants/MQAcceptedPatterns';
+import AggregatorMessageInterface from '../../../interfaces/AggregatorMessageInterface';
+import GlobalCharacterAggregateEntity from '../../../../data/entities/aggregate/global/global.character.aggregate.entity';
+
+@Controller()
+export default class AggregatorGlobalCharacterAggregateController extends AggregatorBaseController {
+    @MessagePattern(MQAcceptedPatterns.GLOBAL_CHARACTER_AGGREGATE)
+    public async process(@Payload() data: AggregatorMessageInterface, @Ctx() context: RmqContext): Promise<void> {
+        try {
+            await this.update(data, context, GlobalCharacterAggregateEntity);
+        } catch (err) {
+            // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access,@typescript-eslint/restrict-template-expressions
+            throw new BadRequestException(`Unable to process message! E: ${err.message}`, MQAcceptedPatterns.GLOBAL_CHARACTER_AGGREGATE);
```

Comment:
Catching and throwing a new error is not really a good practice as it doesn't add anything and only obscures the problems. Besides NestJS knows how to handle the errors.